src/Resources/config/services/factory.xml

```diff
@@ -9,5 +9,6 @@
                  decorates="setono_sylius_redirect.factory.redirect">
             <argument type="service" id="setono_sylius_redirect.custom_factory.redirect.inner" />
         </service>
+        <service id="Setono\SyliusRedirectPlugin\Factory\RedirectFactoryInterface" alias="setono_sylius_redirect.custom_factory.redirect" />
```

Comment:
It should not alias the custom factory should it?

Reply:
That's what Symfony suggested. I guess because that's the service implementing the interface.

Reply:
I am not 100% sure, but I am 90% sure it should be `setono_sylius_redirect.factory.redirect`

Reply:
I am also not sure whether it even matters :D

Reply:
I honestly don't care. The result will be the same one my screen.